### PR TITLE
v1 Contact Us page

### PR DESCRIPTION
**Pull request by ghost**
Following things of note:

- Contact form sends emails using [EmailJS](https://www.emailjs.com/)
- Public keys from EmailJS are safe to save to code directly but code be properly masked on Vercel if we wanted
- Form validation should prevent you from submitting if you do not put in Name and Email. Feel free to try and break this.

### CODE REVIEW

File: app/contact/ContactForm.tsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    return (
+      <div className="w-full mt-0 mb-[15px] mx-0">
+        <form onSubmit={this.sendEmail} ref={this.form}>
+          <label
+            htmlFor="fullNameInput"
+            className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+          >
+            Name
+          </label>
+          <div className="flex">
```
Comment (brandondykun) [inline app/contact/ContactForm.tsx]: Adding mb-6 here will create consistent spacing with other inputs on the form.

**Review by brandondykun — CHANGES_REQUESTED** [review]
Works great, the form cleared when submitted! The last small thing I noticed was the spacing of the inputs. The spacing between the bottom of the name input and the top of the email label is inconsistent with the rest of the form. I added a comment that should fix the issue.

File: app/contact/ContactForm.tsx (hunk 1 above)
Comment (ghost) [reply app/contact/ContactForm.tsx]: Worked like a charm! Thank you so much 🥇

**Review by brandondykun — APPROVED** [review]
Excellent work!

**Pull request merged**